src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Thank you, There were no tests covering this functionality i.e. parsed values returned by hy_split_nevra. I added asserts into tests to check parsed names.
Great, so then I fixed the function.